config/routes.rb

```diff
@@ -265,8 +345,36 @@
       end
   end
 
+  specified_post '/:repository_id/*locid///prove' => 'proofs#create',
+    as: :"theorem_prove",
```

Comment:
Alignment: I would like to leave it as is because it's more readable.
Strings: This is fixed by rebase + commit-amend.